src/components/Navbar.tsx

```diff
@@ -156,7 +156,7 @@ const Navbar = () => {
 									</MenuItem>
 								))}
 							</Menu>
-						</Box> ) : (
+						</Box>) : (
 						<Box>
 							{/* Empty Box since not Admin */}
 						</Box>
```

Comment:
**question:** Is it better to render an empty `Box` or not rendering it altogether? Is there a reason `Box` has to be present on the page even if it's empty?

Reply:
We have opened issues #34 , #32 to address the bugs with the navbar and thank you for pointing these out!